pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/UsingPreparedStatementAndMetaDataObjects/DB_Utils.java

```diff
@@ -0,0 +1,34 @@
+package com.subho.wipro.pjp.tm08.UsingPreparedStatementAndMetaDataObjects;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DB_Utils {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'DB_Utils' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)